src/mapping/epoch-action.ts

```diff
@@ -10,3 +11,30 @@ export function handleEpochUpdateRewards(
   const rewards = BigDecimal.fromString(data.get('rewards')!.toString());
   updatePrice(event, method, receipt, rewards, BigDecimal.zero());
 }
+
+export function handleSuccessfulEpochUnstake(
```

Comment:
```suggestion
export function handleEpochUnstakeSuccess(
```

conciser function name